packages/xo-web/src/xo-app/vm/index.js

```diff
@@ -81,7 +81,8 @@ import VmActionBar from './action-bar'
       pool: getPool(state, props),
       srs: getSrs(state, props),
       vbds: getVbds(state, props),
-      vdis: getVdis(state, props),
+      // Workaround to get VDIs as a self user
```

Comment:
The comment should probably be more specific about the issue since it's not the same problem as usual:
- in one case, the user doesn't have permissions on an object but we still want to get that object
- in this case though, the user does have permissions on the VDI but the permissions cache says that they don't